### ISSUE TITLE
Handle `Not a git repository`

### ISSUE THREAD

**Issue by dex1t** [getsentry/raven-ruby#438]
`Raven::Configuration.new` outputs fatal message on non-git project. This is noisy :cry: 
```
fatal: Not a git repository (or any of the parent directories): .git
```

**Comment by dex1t** [comment]
Thanks for the quick fix :sparkles:

**Comment by nateberkopec** [comment]
Fixed in 0.15.4

**Comment by dex1t** [comment]
:+1:
